spec/unit/ogr/style_manager_spec.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+require 'ogr/style_manager'
+require 'ogr/style_table'
+require 'ogr/style_tool'
+
+RSpec.describe OGR::StyleManager do
+  subject { described_class.create }
+
+  let(:style_table) do
+    st = OGR::StyleTable.create
+    st.add_style('pretty mode', 'LABEL(f:17)')
+    st
```

Comment:
Lately I've been using tap for stuff like this:
```suggestion
    OGR::StyleTable.create.tap do |st|
      st.add_style('pretty mode', 'LABEL(f:17)')
    end
```

Reply:
Oh good idea. I was just pondering if I should allow passing a block to `create`, but this is better.